main.go

```diff
@@ -1,11 +1,37 @@
 package main
 
 import (
-	"github.com/gitlabhq/terraform-provider-gitlab/gitlab"
+	"context"
+	"flag"
+	"log"
+
+	"github.com/gitlabhq/terraform-provider-gitlab/internal/provider"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/plugin"
 )
 
+var (
+	// these will be set by the goreleaser configuration
+	// to appropriate values for the compiled binary
+	version string = "dev"
+)
```

Comment:
Is this hooked up, and if not can you open a new issue for setting the appropriate ldflags during release?

Reply:
yes, it already was: https://github.com/gitlabhq/terraform-provider-gitlab/blob/6c1a60d79fc76012287e21696384c3c019639ed2/.goreleaser.yml#L8